iclr2017/process/commentProcess.js

```diff
@@ -67,12 +67,36 @@ function(){
       return or3client.or3request(or3client.grpUrl+'?id=ICLR.cc/2017/pcs',{},'GET',token)
       .then(result=>{
         var pcs = result.groups[0].members;
+
+        console.log('pcs before filter: '+pcs);
+
+        var workshopIdx = pcs.indexOf('ICLR.cc/2017/workshop');
+        if(workshopIdx>-1){
```

Comment:
Please, start to follow the js guidelines: http://standardjs.com/rules.html. Spaces after keywords.